inscriptions/ISic000858.xml

```diff
@@ -185,16 +187,29 @@
             <graphic n="print" url="ISic000858.jpg" height="3680px" width="5520px">
                 <desc>I.Sicily with the permission of the Assessorato Regionale dei Beni Culturali e dell’Identità Siciliana - Dipartimento dei Beni Culturali e dell’Identità Siciliana</desc>
             </graphic>
-         </surface>
+        </surface>
     </facsimile>
     <text>
         <body>
             <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/><orig>ΞΑΡΧ</orig><gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="2"/><expan><abbr>Κ</abbr><ex cert="low">υίντος</ex></expan> <g ref="#interpunct">·</g> Λόλλι<supplied reason="lost" cert="low">ος</supplied>
-                    <lb n="3"/><orig>Δ</orig> <g ref="#interpunct">·</g> <orig>ΠΟ</orig><gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="4"/><orig>ΧΜΑΡ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="1"/><orig n="5">ΞΑΡΧ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><w n="10"><expan><abbr>Κ</abbr><ex cert="low">υίντος</ex></expan></w> <g ref="#interpunct" n="15">·</g> <w n="20">Λόλλι<supplied reason="lost" cert="low">ος</supplied></w>
+                    <lb n="3"/><orig n="25">Δ</orig><g ref="#interpunct" n="30">·</g> <orig n="35">ΠΟ</orig><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
not crucial, but I see that it closed up a space here between the first orig element and the following g interpunct